app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<h2 class="m-3 d-flex justify-content-center">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field m-3 d-flex justify-content-center">
+    <%= f.text_field :name, placeholder: "Username", class: "w-50" %>
+  </div>
+
+  <div class="field m-3 d-flex justify-content-center">
+    <%= f.text_field :photo, placeholder: "Photo", class: "w-50" %>
+  </div>
+
+  <div class="field m-3 d-flex justify-content-center">
+    <%= f.text_field :bio, placeholder: "Bio", class: "w-50" %>
+  </div>
```

Comment:
[OPTIONAL] I think it's better to remove the bio and the photo from the registration. Let s keep only the name, email, password, and password confirmation 😊.